enterprise/migrations/0137_enrollment_email_update.py

```diff
@@ -0,0 +1,230 @@
+                                    </tr>
+                                </tbody>
+                            </table>
+                        </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </td>
+            </tr>
+            <tr>
+            <td>
+                <table cellpadding="0" cellspacing="0" width="100%">
+                    <tbody>
+                        <tr>
+                        <td height="20" style="line-height:1px;font-size:1px"></td>
+                        </tr>
+                        <tr>
+                        <td align="left" valign="top"><a href="https://business.edx.org" style="font-family:'Open Sans',Arial,sans-serif;font-size:14px;line-height:17px;text-decoration:none;color:#707070" target="_blank"><span style="color:#00262b">edX for Business</span><span style="color:#707070"> — eLearning Solutions for Your Company</span></a></td>
+                        </tr>
+                        <tr>
+                        <td height="20" style="line-height:1px;font-size:1px"></td>
+                        </tr>
+                        <tr>
+                        <td align="left" style="font-family:'Open Sans',Arial,sans-serif;color:#707070;font-size:14px;line-height:17px" valign="top">© 2021 edX Inc. All rights reserved.</td>
+                        </tr>
+                        <tr>
+                        <td height="20" style="line-height:1px;font-size:1px"></td>
+                        </tr>
+                        <tr>
+                        <td height="20" style="line-height:1px;font-size:1px"></td>
+                        </tr>
+                        <tr>
+                        <td align="left" style="font-family:'Open Sans',Arial,sans-serif;color:#707070;font-size:14px;line-height:17px" valign="top">141 Portland St. 9th Floor, Cambridge, MA 02139</td>
+                        </tr>
+                        <tr>
+                        <td height="30" style="line-height:1px;font-size:1px"></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </td>
+            </tr>
+        </tbody>
+    </table>
+</body>
+</html>
+         """},
+    )
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0136_auto_20210629_2129'),
+    ]
+
+    operations = [
+        migrations.RunPython(code=update_email, reverse_code=update_email)
```

Comment:
The reason I have the reverse_code calling the same method is that I didn't see the need to paste the old HTML in this one, because no functionality will change based up if this one is migrated back. However, I didn't want a migration that could be rolled back.